.circleci/config.yml

```
@@ -137,6 +137,7 @@ commands:
             mkdir -p /tmp/junit/stack
             stack --no-terminal --stack-yaml << parameters.stack_yaml_file >> test -j1 liquidhaskell:test << parameters.extra_build_flags >> << parameters.extra_test_flags >> --ta="--liquid-runner \"<< parameters.liquid_runner >>\"" --ta="-t 1200s --xml=/tmp/junit/stack/main-test-results.xml": #--liquid-opts='--cores=1'":
             stack --no-terminal --stack-yaml << parameters.stack_yaml_file >> test -j1 liquidhaskell:liquidhaskell-parser << parameters.extra_build_flags >> --ta="--xml=/tmp/junit/stack/parser-test-results.xml":
+            stack --no-terminal  --stack-yaml << parameters.stack_yaml_file >> test -j1 stitch-lh << parameters.extra_build_flags >>
```

```suggestion
            # Testing stitch-lh to catch regressions in the verification
            stack --no-terminal  --stack-yaml << parameters.stack_yaml_file >> test -j1 stitch-lh << parameters.extra_build_flags >>
```
Perhaps this extra test of stitch-lh, and later the benchmarks could be run in separate CI steps. e.g.
```
- run:
    name: Verify stitch-lh
    command: |
      # Testing stitch-lh to catch regressions in the verification
      stack --no-terminal  --stack-yaml << parameters.stack_yaml_file >> test -j1 stitch-lh << parameters.extra_build_flags >>
```

---

Changed.